### PR TITLE
Fix aliasing properly.

### PR DESCRIPTION
**Pull request by TAG-Epic**
## Summary
Aliasing should now be fixed, and should be able to be disabled via the nocompat extra

## Checklist

TODO:
- [x] implement automatic testing
- [ ] disabling via the nocompat extra
- [ ] compat checking with libraries such as jishaku, dislash etc

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by TAG-Epic** [comment]
Issue found where the OS hasn't finished writing before the test starts... If there's a error when building the alias, just run it again.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015-present Rapptz
+Copyright (c) 2021-present tag-epic
```
Comment (veni-vidi-code) [inline LICENSE:4]: I am not sure if you are allowed to take him out commpletly. Most of the code is still from his repo

**Comment by ooliver1** [comment]
Hasn't been taken out. Epic just added

File: LICENSE (hunk 1 above)
Comment (TAG-Epic) [reply LICENSE:4]: Danny's line wasn't removed here????

Comment (ooliver1) [reply LICENSE:4]: Hasn't been taken out. Epic just added

Comment (veni-vidi-code) [reply LICENSE:4]: my bad. I should switch to my pc, reviewing from the phone is hard. sry

File: alias_license.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-present Rapptz
```
[the remark this answers is not in the document]
Comment (ooliver1) [reply alias_license.py:4]: He has been the owner of 99% of the code, why not just leave it

[the remark this answers is not in the document]
Comment (TAG-Epic) [reply alias_license.py:4]: No this is just for the alias which already isnt under dannys copyright?

[the remark this answers is not in the document]
Comment (veni-vidi-code) [reply alias_license.py:4]: I do not think the MIT License allows you to taake him out. Maybe ask him about his opinion

[the remark this answers is not in the document]
Comment (ooliver1) [reply alias_license.py:4]: No harm keeping him in lmao

**Comment by veni-vidi-code** [comment]
By whom shall this be run? By the devs or by an other script?

(This could be interestig in order of adding possible debug messages when its finished and more)

**Comment by TAG-Epic** [comment]
> By whom shall this be run? By the devs or by an other script?
> 
> (This could be interestig in order of adding possible debug messages when its finished and more)

Hmmmm probably on install to not pollute the git history

**Comment by veni-vidi-code** [comment]
> > By whom shall this be run? By the devs or by an other script?
> > (This could be interestig in order of adding possible debug messages when its finished and more)
> 
> Hmmmm probably on install to not pollute the git history, and allow for the nocompat extra

Maybe add it to setup.py so it gets build when the package gets build? By that we do not get problems with antiviruses that might not like this behaviour?

**Comment by TAG-Epic** [comment]
> > > By whom shall this be run? By the devs or by an other script?
> > > (This could be interestig in order of adding possible debug messages when its finished and more)
> > 
> > 
> > Hmmmm probably on install to not pollute the git history, and allow for the nocompat extra
> 
> Maybe add it to setup.py so it gets build when the package gets build? By that we do not get problems with antiviruses that might not like this behaviour?

yeah thats what I was thinking too.

**Comment by TAG-Epic** [comment]
Hmmm automatic testing should probably be done in another pr.

**Review by The-Bow-Hunter — CHANGES_REQUESTED** [review]
The automated created files should not be on github but rather be automated by setup.py in my opinion

See https://github.com/nextcord/nextcord/pull/151#issuecomment-914461372

**Pull request merged**